crates/hdk/src/hdk.rs

```diff
@@ -9,10 +9,10 @@ pub const HDK_NOT_REGISTERED: &str = "HDK not registered";
 /// Every test needs its own mock so each test needs to set it.
 use core::cell::RefCell;
 
-#[cfg(feature = "mock")]
+#[cfg(any(feature = "mock", not(target_arch = "wasm32")))]
```

Comment:
why do suppose this is necessary?

Reply:
I get `\hdk.dll : fatal error LNK1120: 35 unresolved externals`.

Host functions don't resolve at link by default:
`hdk.hdk.c79tk70i-cgu.8.rcgu.o : error LNK2019: unresolved external symbol __hash_entry referenced in function 
_ZN52_$LT$hdk..hdk..HostHdk$u20$as$u20$hdk..hdk..HdkT$GT$10hash_entry17hb265ccef3ccfb5c3E`

Maybe there is some link config to change but I found it easier to just exclude HostHdk.